modules/ROOT/pages/rtfctl-permissions.adoc

```diff
@@ -0,0 +1,112 @@
+= RTFCTL Kubernetes Permissions
+
+Understanding and enabling the specific permissions required over the Kubernetes resources is crucial for successfully running `rtfctl` commands. Properly configured role permissions are essential for seamless execution of `rtfctl` operations.
+
+The following table lists all the permissions that you configure using K8s RBACs (https://kubernetes.io/docs/reference/access-authn-authz/rbac/[Role based access control^]) in Kubernetes: 
+
+* Namespace +
+The Namespace column indicates that the permission can be a Role based or ClusterRole based:
```

Comment:
Should Namespace be bold, since it's a UI element?

Why is Role init cap? Hyphenate compound adjectives. 

Is ClusterRole a UI element?